tests/confest.py

```
@@ -0,0 +1,39 @@
+    async with async_session() as session:
+        yield session
+    
+    async with engine.begin() as conn:
+        await conn.run_sync(Base.metadata.drop_all)
+
+@pytest.fixture(scope="function")
+async def client(db_session):
+    async def override_get_db():
+        yield db_session
+
+    app.dependency_overrides[get_db] = override_get_db
```

Ruff

Undefined name `get_db`

(F821)